lib/core/src/Cardano/Wallet/Api/Server.hs

```diff
@@ -354,7 +356,7 @@ instance LiftHandler ErrCoinSelection where
                 [ "I can't process this payment because there's not enough "
                 , "UTxO available in the wallet. The total UTxO sums up to "
                 , showT utxo, " Lovelace, but I need ", showT payment
-                , " Lovelace (inclusive of fee amount) in order to proceed "
+                , " Lovelace (exclusive of fee amount) in order to proceed "
```

Comment:
:man_shrugging:

Reply:
In this case, I would recommend "excluding fee amount".

Reply:
Fixed :+1: